examples/lock-app/silabs/src/LockManager.cpp

```diff
@@ -18,16 +18,24 @@
  */
 
 #include "LockManager.h"
-
 #include "AppConfig.h"
 #include "AppTask.h"
 #include <app-common/zap-generated/attributes/Accessors.h>
 #include <cstring>
 #include <lib/support/logging/CHIPLogging.h>
 
+using chip::app::DataModel::MakeNullable;
 using namespace ::chip::DeviceLayer::Internal;
+using namespace EFR32DoorLock;
 using namespace EFR32DoorLock::LockInitParams;
 
+static uint16_t LockUserInfoSize               = sizeof(LockUserInfo);
+static uint16_t LockCredentialInfoSize         = sizeof(LockCredentialInfo);
+static constexpr uint16_t CredentialStructSize = sizeof(CredentialStruct);
+static uint16_t WeekDayScheduleInfoSize        = sizeof(WeekDayScheduleInfo);
+static uint16_t YearDayScheduleInfoSize        = sizeof(YearDayScheduleInfo);
+static uint16_t HolidayScheduleInfoSize        = sizeof(HolidayScheduleInfo);
```

Comment:
Those can be all const expr. its how you use them that must change
see my comment here
https://github.com/project-chip/connectedhomeip/pull/37849#discussion_r1995740967